.travis.yml

```diff
@@ -64,6 +64,8 @@ jobs:
         - gem install mdl
         - pip install --user --upgrade pip
         - pip install --user yamllint
+        - pip install --user pylint
```

Comment:
it seems that on Ubuntu you can install this with `apt-get` instead, that is cleaner in most cases:
```
apt-get install python3-pylint
```
and that would also get you the python3 version (no more warnings?)